mei40To50/mei40To50.xsl

```diff
@@ -231,19 +306,114 @@
         </xsl:if>
     </xsl:template>
     
+    <xd:doc>
```

Comment:
```suggestion
    
    <xd:doc>
        <xd:desc>
            <xd:p>Removes @visible on mRest</xd:p>
        </xd:desc>
    </xd:doc>
    <xsl:template match="mei:mRest/@visible">
        <xsl:if test="$verbose">
            <xsl:message select="'Dropping @visible on mRest ' || ancestor-or-self::mei:*[@xml:id][1]/@xml:id"/>
        </xsl:if>
    </xsl:template>
    
    <xd:doc>
```

Reply:
Proposal to go with this order:

- Removes @visible on mRest
- Removes @instr on mRest
- Removes @instr on mSpace
- Removes @instr on multiRest
- Removes @instr on rest

Reply:
fixed offline submitted via commit

Reply:
please check